src/glossary.rst

```diff
@@ -7685,6 +7685,30 @@ visibility modifier
 A :dt:`visibility modifier` sets the :t:`visibility` of the :t:`name` of an
 :t:`item`.
 
+.. _fls_dLlUt8PrXAls:
+
+visible emptiness
+^^^^^^^^^^^^^^^^^
+
+:dp:`fls_shXDYqnUy2Pb`
+A :dt:`visible empty type` is a :t:`type` subject to :t:`visible emptiness`.
```

Comment:
text in the wrong place... should go to `visible empty type` section

other text in the 2 sections that follow is also in wrong place